src/System.Security.Cryptography.Xml/tests/SignedXmlTest.cs

```diff
@@ -284,7 +284,7 @@ public void AsymmetricRSAMixedCaseAttributesVerifyWindows()
             Assert.True(v1.CheckSignature());
         }
 
-        [Fact(Skip = "https://github.com/dotnet/corefx/issues/16691")]
+        [Fact]
         public void AsymmetricDSASignature()
```

Comment:
You and I are setting up for an inner-loop test fail due to merging.  Do you mind slapping `[ActiveIssue(17001, TestPlatforms.OSX)]` on here?  When https://github.com/dotnet/corefx/pull/17011 merges macOS loses the ability to generate DSA keys.